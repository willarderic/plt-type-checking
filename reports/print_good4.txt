TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: PRINT  LEXEME: print					TOKEN: PRINT     LEXEME: print
TOKEN: T  LEXEME: t						TOKEN: T         LEXEME: t
CONST -> t							CONST -> t
EXPR -> CONST							EXPR -> CONST
PRINT_EXPR -> print EXPR					PRINT_EXPR -> print EXPR
PARENTHESIZED_EXPR -> PRINT_EXPR				PARENTHESIZED_EXPR -> PRINT_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

