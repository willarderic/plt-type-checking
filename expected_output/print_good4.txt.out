TOKEN: LPAREN    LEXEME: (
TOKEN: PRINT     LEXEME: print
TOKEN: T         LEXEME: t
CONST -> t
EXPR -> CONST
PRINT_EXPR -> print EXPR
PARENTHESIZED_EXPR -> PRINT_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR

---- Completed parsing ----

